### PR TITLE
📝 Edited SLDs to be valid XMLs

### PR DESCRIPTION
**Pull request by Jeremy-Prior**
Some of the SLDs had additional '<se:' in the code so could not be used as styles on the geonode.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 0 sec

File: styles/ecowas_power_plant_points_gcs_wgs84_wapp_pre_delete_20230609.sld [hunk 1]
```diff
@@ -58,12 +58,12 @@
...
               <!--Parametric SVG-->
               <se:ExternalGraphic>
-              <se:<OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>
+               <OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>
                 <se:Format>image/svg+xml</se:Format>
               </se:ExternalGraphic>
               <!--Plain SVG fallback, no parameters-->
               <se:ExternalGraphic>
-              <se:<OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>   
+               <OnlineResource xlink:type="simple" xlink:href="file:///opt/geoserver/data_dir/styles/svg/powerplants_hydro.svg"/>
```
Comment (NyakudyaA) [inline styles/ecowas_power_plant_points_gcs_wgs84_wapp_pre_delete_20230609.sld:66]: Can you change the path from `file:///opt/geoserver/data/` to `file:///geoserver_data/data/`

**Comment by NyakudyaA** [comment]
@Jeremy-Prior Did you validate them all in GeoServer?

**Comment by Jeremy-Prior** [comment]
@NyakudyaA I tested these ones on the GeoNode instance when the previous ones wouldn't upload? I can't get into the GeoServer?

File: styles/ecowas_power_plant_points_gcs_wgs84_wapp_pre_delete_20230609.sld (hunk 1 above)
Comment (Jeremy-Prior) [reply styles/ecowas_power_plant_points_gcs_wgs84_wapp_pre_delete_20230609.sld]: Done

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 56 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 58 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 0 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 59 sec

**Pull request merged**